configs/_base_/datasets/kitti-mono3d.py

```diff
@@ -0,0 +1,99 @@
+dataset_type = 'KittiMonoDataset'
+data_root = 'data/kitti/'
+class_names = ['Pedestrian', 'Cyclist', 'Car']
+# Input modality for kitti dataset, this is consistent with the submission
+# format which requires the information in input_modality.
+input_modality = dict(
+    use_lidar=False,
+    use_camera=True,
+    use_radar=False,
+    use_map=False,
+    use_external=False)
```

Comment:
I have simplified some params here, please refer to configs [here](https://github.com/open-mmlab/mmdetection3d/pull/964/files) and make them consistent.